src/indev/lv_indev.h

```diff
@@ -113,31 +117,81 @@ lv_indev_t * lv_indev_active(void);
  */
 void lv_indev_set_type(lv_indev_t * indev, lv_indev_type_t indev_type);
 
+/**
+ * Set a function pointer to read input device data to the indev
+ * @param indev pointer to an input device
+ * @param read_cb pointer to function pointer to read input device data
+ */
 void lv_indev_set_read_cb(lv_indev_t * indev,  lv_indev_read_cb_t read_cb);
 
+/**
+ * Set user data to the indev
+ * @param indev pointer to an input device
+ * @param user_data pointer to user data
+ */
 void lv_indev_set_user_data(lv_indev_t * indev, void * user_data);
 
+/**
+ * Set driver data to the indev
+ * @param indev pointer to an input device
+ * @param driver_data pointer to driver data
+ */
 void lv_indev_set_driver_data(lv_indev_t * indev, void * driver_data);
 
+/**
+ * Assign a display to the indev
+ * @param indev pointer to an input device
+ * @param disp pointer to an display
+ */
+void lv_indev_set_display(lv_indev_t * indev, struct _lv_display_t * disp);
+
 /**
  * Get the type of an input device
  * @param indev pointer to an input device
  * @return the type of the input device from `lv_hal_indev_type_t` (`LV_INDEV_TYPE_...`)
  */
 lv_indev_type_t lv_indev_get_type(const lv_indev_t * indev);
 
+/**
+ * Get a pointer to the function to read input device data to the indev
+ * @param indev pointer to an input device
+ * @return Pointer to function pointer to read input device data or NULL if indev is NULL
```

Comment:
_function pointer_ -> _callback function_

Reply:
Done in 25f295f